go.mod

```diff
@@ -1,28 +1,26 @@
 module github.com/kubevirt/vm-console-proxy
 
-go 1.21
+go 1.22.4
 
 require (
 	github.com/emicklei/go-restful/v3 v3.12.0
 	github.com/fsnotify/fsnotify v1.7.0
 	github.com/golang-jwt/jwt/v4 v4.5.0
 	github.com/golang/mock v1.6.0
-	github.com/onsi/ginkgo/v2 v2.17.1
-	github.com/onsi/gomega v1.32.0
-	github.com/openshift/api v0.0.0-20240111203005-e04828eaf17a // release-4.15
-	github.com/openshift/library-go v0.0.0-20240111120724-db0c36cd2185 // release-4.15
-	k8s.io/api v0.29.3
-	k8s.io/apimachinery v0.29.3
-	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/utils v0.0.0-20240310230437-4693a0247e57
-	kubevirt.io/api v1.2.0
-	kubevirt.io/client-go v1.2.0
+	github.com/onsi/ginkgo/v2 v2.17.2
+	github.com/onsi/gomega v1.33.1
+	github.com/openshift/api v0.0.0-20240702171116-4b89b3a92a17 // release-4.16
```

Comment:
In https://github.com/openshift/release/pull/54527 you upgraded to 4.17. Do that here as well?

Reply:
done